types/yup/yup-tests.ts

```diff
@@ -594,3 +599,41 @@ yup.object<MyInterface>({
         .required(),
     arrayField: yup.array(yup.string()).required()
 });
+
+const personSchema = yup.object({
+    firstName: yup.string(), // $ExpectType StringSchema<string>
+    email: yup
+        .string()
+        .nullable()
+        .email(),
+    birthDate: yup
+        .date()
+        .nullable()
+        .min(new Date(1900, 0, 1)),
+    canBeNull: yup.string().nullable(true), // $ExpectType StringSchema<string | null>
```

Comment:
Maybe have a boolean test case as well since we're adding types for `BooleanSchema` too?

Reply:
👍